### PR TITLE
COL-1222 First-round Activity Network implementation

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-1222

Refinements to come.

screen shot 2017-11-09 at 11 21 22 am

### CODE REVIEW

File: public/app/dashboard/activityNetworkTooltip.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      </td>
+    </tr>
+  </table>
+  <div>
+    <button type="button" class="btn btn-link" data-ng-click="switchUser(selectedUser)">
+      View <span data-ng-bind="selectedUser.canvas_full_name"></span>'s profile
+    </button>
+  </div>
+  <div>
+    Last activity: July 14, 1789
```
Comment (paulkerschen) [inline public/app/dashboard/activityNetworkTooltip.html]: This file is obviously the least fleshed out of the bunch.

**Comment by paulkerschen** [comment]
"Add Assets" in the screenshot is outdated; in code the checkbox reads "Use Assets."

File: public/app/dashboard/activityNetworkDirective.js [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+              .attr('y', function(d) { return d.y; });
+          };
+
+          sizeAndRestart();
+        });
+      }
+    };
+  });
+
+}(window.angular));
```
Comment (johncrossman) [inline public/app/dashboard/activityNetworkDirective.js]: 377 lines of goodness!

**Comment by johncrossman** [comment]
The Activity Network's blank area speaks volumes. First round looks good, @paulkerschen

**Pull request merged**